lib/src/app_credentials.dart

```diff
@@ -0,0 +1,49 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+import 'native/realm_core.dart';
+
+/// An enum containing all authentication providers. These have to be enabled manually for the application before they can be used
+/// your application before they can be used.
```

Comment:
```suggestion
/// An enum containing all authentication providers. These have to be enabled manually for the application before they can be used.
```

There's some repetition here.